JhipsterBlazor/Shared/Constants/ErrorConst.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace JhipsterBlazor.Shared.Constants
+{
+    public static class ErrorConst
```

Comment:
Maybe we can create another project for const / exceptions etc. The goal is to shared this part with backend